### ISSUE TITLE
RSS Feed InvalidHttpGUID Error

### ISSUE THREAD

**Issue by Jan-Ka** [worldanvil/worldanvil-bug-tracker#593]
**World Anvil Username**:
ka_jan

**World / Campaign / RPG Character / Manuscript Title**:
https://www.worldanvil.com/w/inquest

**Feature**: 
RSS Feed

**Related URLS** (To public & visible pages add screenshots otherwise):
https://www.worldanvil.com/w/inquest/opendata/rss
https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.worldanvil.com%2Fw%2Finquest%2Fopendata%2Frss

**Describe the Issue**
RSS Feed Validation fails on `guid` field with InvalidHttpGUID Error, proposes Change [InvalidHttpGUID](https://validator.w3.org/feed/docs/error/InvalidHttpGUID.html); adding `isPermaLink="false"` to guid tag.

**To Reproduce**
1. Copy World RSS Feed Link
2. Open [](https://validator.w3.org/feed/)
3. Paste World RSS Feed Link
4. Click "Check"

World needs to have recently updated articles.

**Expected behavior**
W3C RSS Feed Validator reports valid Feed

**Screenshots**
![image](https://user-images.githubusercontent.com/11042838/134822090-52a33be5-c57a-4d5d-8fa9-64a7ce71b0dc.png)

**Comment by QuibblingAsh42** [comment]
Confirming this is happening to my feeds, too.  I had found it when I tried to setup an action with IFTTT, and it never triggered.  Their help page suggested validating, and mine fails just like Jan-Ka's

https://help.ifttt.com/hc/en-us/articles/115010396768-My-RSS-feed-isn-t-working-or-updating

Unfortunately, I don't have any more detail beyond "it doesn't work" because I don't have any access to logs from IFTTT.  I can confirm it DOES fire Microsoft Power Automate flows just fine, so it might only be an issue for extra-picky consuming software.
